winterfell/src/lib.rs

```diff
@@ -111,10 +111,10 @@
 //! };
 //!
 //! pub fn build_do_work_trace(start: BaseElement, n: usize) -> TraceTable<BaseElement> {
-//!     // Instantiate the trace with a given width and length; this will allocate all
-//!     // required memory for the trace
+//!     // Instantiate the trace with a given widths, length and number of RAP coins;
```

Comment:
I think should be "a given width"

Reply:
rather "with given widths", given that we specify both original and rap width. But I'll reformulate